packages/gas-station/src/abi/gas-station.ts

```diff
@@ -152,26 +158,6 @@ export const gasStationAbi = [
     stateMutability: "view",
     type: "function",
   },
-  {
-    inputs: [
```

Comment:
should we make explicit note of this somewhere? or no need given this wasn't really being used in the first place?